bool active
bool failure
float32 total_duration
float32 distance_to_robot
int32[2] virtual_leader_cell # [x, y]
